01week/rockPaperScissors.js

```diff
@@ -30,7 +30,6 @@ function rockPaperScissors(hand1, hand2) {
 		return "Hand one wins!";
 	}
 }
```

Comment:
the logic here works. but i hope you can see how it can be "shorten" or less repetitive!